.github/workflows/build.yml

```diff
@@ -0,0 +1,20 @@
+name: ci
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
```

Comment:
We should use commit hashes such as [this example](https://github.com/nanocurrency/nano-work-server/blob/master/.github/workflows/build.yml#L28)

Reply:
Done